packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -54,13 +54,12 @@ class KnexAdapter extends BaseKeystoneAdapter {
     const createResult = await pSettle(
       Object.values(this.listAdapters).map(listAdapter => listAdapter.createTable())
     );
+    const errors = createResult.filter(({ isRejected }) => isRejected).map(({ reason }) => reason);
 
-    const errors = createResult.filter(({ isRejected }) => isRejected);
     if (errors.length) {
       if (errors.length === 1) throw errors[0];
       const error = new Error('Multiple errors in KnexAdapter.postConnect():');
-      error.errors = errors.map(({ reason }) => reason);
-      throw error;
+      throw { ...error, errors };
```

Comment:
Spreading like this changes it from a type of `Error` to a type of `Object` which can break other things like error checking code in the Apollo stack.

```suggestion
      error.errors = errors;
      throw error;
```